pulsar-testclient/src/main/java/org/apache/pulsar/testclient/DefaultMessageFormatter.java

```diff
@@ -89,8 +90,7 @@ private String getFloatValue(float size) {
             return String.valueOf(r.nextFloat());
         }
         String format = "%" + size + "f";
-
-        return String.format(format, get_FloatValue(size));
+        return String.format(Locale.US, format, get_FloatValue(size));
```

Comment:
When I ran the tests on my machine the floating point numbers were being formatted with a comma instead of a decimal. It might have something to do with my machine having the default locale on the system as Europe, so I thought it would be best to make sure that US Locale is used, as JSON strings won't work with , separated floating point numbers.
I am open to other suggestions, or reverting this.

Reply:
I suggest we use the default setting.